supertokens_python/recipe/dashboard/recipe_implementation.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Dict
+
+from supertokens_python.framework import BaseRequest
+from .interfaces import (
+    RecipeInterface,
+)
+from supertokens_python.constants import DASHBOARD_VERSION
+from .utils import DashboardConfig
+
+if TYPE_CHECKING:
+    from supertokens_python.querier import Querier
+
+
+class RecipeImplementation(RecipeInterface):
+    def __init__(self, querier: Querier):
```

Comment:
querier is not used in this recipe impl.

Reply:
removed.